services/galley/src/Galley/Data/Queries.hs

```diff
@@ -39,11 +40,11 @@ selectTeamConv = "select managed from team_conv where team = ? and conv = ?"
 selectTeamConvs :: PrepQuery R (Identity TeamId) (ConvId, Bool)
 selectTeamConvs = "select conv, managed from team_conv where team = ? order by conv"
 
-selectTeamMember :: PrepQuery R (TeamId, UserId) (Identity Permissions)
-selectTeamMember = "select perms from team_member where team = ? and user = ?"
+selectTeamMember :: PrepQuery R (TeamId, UserId) (Identity (Permissions, Maybe UserId, Maybe UTCTimeMillis))
```

Comment:
No need to use `Identity` here; that's only necessary when retrieving a single field